chrome/updater/updater_module.cc

```diff
@@ -330,9 +330,9 @@ void UpdaterModule::Update() {
 
 // The following methods are called by other threads than the updater_thread_.
 
-void UpdaterModule::CompareAndSwapChannelChanged(int old_value, int new_value) {
+void UpdaterModule::CompareAndSwapForcedUpdate(int old_value, int new_value) {
   auto config = updater_configurator_;
-  if (config) config->CompareAndSwapChannelChanged(old_value, new_value);
+  if (config) config->CompareAndSwapForcedUpdate(old_value, new_value);
```

Comment:
```
  if (config) {
    config->CompareAndSwapForcedUpdate(old_value, new_value);
  }
```

Reply:
I found the following reference from the style guide:
```
For historical reasons, we allow one exception to the above rules: the curly braces for the controlled statement or the line breaks inside the curly braces may be omitted if as a result the entire statement appears on either a single line (in which case there is a space between the closing parenthesis and the controlled statement) or on two lines (in which case there is a line break after the closing parenthesis and there are no braces).

// OK - fits on one line.
if (x == kFoo) { return new Foo(); }

// OK - braces are optional in this case.
if (x == kFoo) return new Foo();

// OK - condition fits on one line, body fits on another.
if (x == kBar)
  Bar(arg1, arg2, arg3);
```
https://engdoc.corp.google.com/eng/doc/devguide/cpp/styleguide.md?cl=head#Formatting_Looping_Branching

I agree that adding braces improves readability, and I'll modify all places in this file that has a single-line controlled statement without braces. But I'd like to note that this style (single-line without braces) is prevalent in our code base, and should be fixed systematically if we think it's important.

Reply:
I'm OK with doing a round of clang-format in a follow-up CL (with no functional changes, so it's easier to review and land).

Re. the argument about this style being pervasive, while it's true that the folder style should be respected, this file belong to `//chrome/updater`, which is technically a Chrome folder. Are there many examples of this in-line no-braces style in this folder?

Reply:
This folder was previously called `//cobalt/updater` and was renamed to `//chrome/updater` with this cl: https://lbshell-internal-review.git.corp.google.com/c/cobalt_src/+/286760. The code under this folder was mostly contributed by Cobalt engineers to customize the updater for Cobalt. In terms of single-line statement without braces, there're references under this folder, e.g. 
1. https://source.corp.google.com/h/lbshell-internal/cobalt_src/+/25.lts.1+:chrome/updater/prefs.cc;l=25;bpv=1
2. https://source.corp.google.com/h/lbshell-internal/cobalt_src/+/25.lts.1+:chrome/updater/installer.cc;l=92;bpv=0
3. https://source.corp.google.com/h/lbshell-internal/cobalt_src/+/25.lts.1+:chrome/updater/win/util.cc;l=121
etc

The best way to solve this is to do a follow-up CL across the code base to fix the format. It's good to do it with Chrobalt, because this 25.lts.1+ branch will be deprecated anyway.

Reply:
Hmmkay, the argument of this being a maintenance branch makes sense. 

I guess I was confused as to the nature of //chrome/updater being a cobalt creature, since there's a similarly named folder in Chrome (https://source.chromium.org/chromium/chromium/src/+/main:chrome/updater/) and I saw chromium (not cobalt) file headers.

Reply:
The folder was originally pulled from Chrome, as many Cobalt folders were. Then it was heavily modified/customized by Cobalt engineers to suit the needs of Cobalt, as many Cobalt folders were. With Chrobalt, this folder will be updated, and it'll be a good time to normalize the style.